### PR TITLE
Fix minor english in the Google Service Account section

### PR DESCRIPTION
**Pull request by ukd1**
1. correct ``where <gsa_json_file_path> must the GSA credentials in JSON form`` as it is missing a word
2. remove link to ticket (none of the other sections have it)

### CODE REVIEW

**Comment by odeke-em** [comment]
Hello @ukd1, thank you for the fix and welcome to drive!
My apologies for the late reply, I've swamped with a bunch of work but thanks to @jean-christophe-manciot who helped bring it to my attention.

File: README.md [hunk 1]
```diff
@@ -185,10 +185,7 @@ drive init --service-account-file <gsa_json_file_path> ~/gdrive
 cd ~/gdrive
 ```
 
-where <gsa_json_file_path> must the GSA credentials in JSON form.
-This feature was implemented as requested by:
-+ https://github.com/odeke-em/drive/issues/879
-
+Where <gsa_json_file_path> must a [Google Service Account credentials](https://developers.google.com/identity/protocols/OAuth2ServiceAccount#creatinganaccount) file in JSON form.
```
Comment (odeke-em) [inline README.md]: Minor nit:
s/must a/must be a/g

**Comment by idvoretskyi** [comment]
Due to a silence in this thread, I've created https://github.com/odeke-em/drive/pull/997

**Comment by odeke-em** [comment]
@ukd1 thank you for working on this issue! @idvoretskyi made a PR in https://github.com/odeke-em/drive/pull/997 and gave you credit for it.

**Pull request closed without merging**